back_end/README.md

```diff
@@ -0,0 +1,20 @@
+# Back End Challenge
+
+Your challenge is to create a simple API application using Ruby on Rails (or the web development framework of your choice). Being a SVOD service, we have film data that we need to surface through an API. Your challenge is build a small sample endpoint for retrieving film data and submitting film ratings.
+
+The application should provide the following functionality:
+- RESTful endpoints for retrieving a list of films or an individual film
+- Support for allowing a user to rate a specific film
+- Include average rating for a film as one of its attributes
+- Requests and responses should follow the [JSON API specification](http://jsonapi.org/)
+
+Please see [films.json](films.json) for sample film data.
+
+We don't expect you to have a perfect solution in the time allotted. Do as much as is reasonable to demonstrate your technical abilities. Please submit your work by sending us a link to a GitHub repo with your code.
+
+Bonus points for the following:
+- Support pagination
+- Support filtering/sorting
+- Support for only retrieving the attributes as listed in the request
+- Document the end points (options that can be passed, etc)
+- Provide all the documentation we'd need to run the application
```

Comment:
I'd rather have more code and not be able to run it than less code and documentation, personally. This is also in the bonus section in one of the other challenges. How strongly do you feel? :)